app/views/citizens/additional_accounts/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<% content_for :navigation do %>
+  <%#= link_to 'Back', path-to-be-determined, class: 'govuk-back-link', id: 'back' %>
+<% end %>
+
+  <%= form_tag(citizens_additional_account_path(id: :update), method: :patch) do %>
```

Comment:
**[suggestion/not a blocker]** If we were conforming with the conventions associated with the form template name and the associated RESTFUL action, this should probably be a **post**.

Reply:
It could be a post, but then I'd have to either customise the routes or put the second action into another controller. I could argue that the resource is the citizen's decision on whether to add additional accounts, and this action is a modification of that decision.